### DISCUSSION TITLE
Segnalazione Riparazioni dopo aggiornamento

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#2568] [HUB: Home Assistant]
Questa mattina ho fatto l'ultimo aggiornamento ad HA,  mi sono trovato due avvisi di riparazioni relative a MQTT che onestamente non ho capito.
Questa parte di una delle due:

**
Alcune entità MQTT hanno un nome di entità che inizia con il nome del dispositivo. Questo non è previsto. Per evitare un nome duplicato, il prefisso del nome del dispositivo viene privato del nome dell'entità come soluzione alternativa. Si prega di informare il manutentore dell'applicazione software che fornisce le entità interessate per risolvere questo problema.

Elenco delle entità interessate:

    switch.test_energia_power_alarm
    sensor.telecomando_scrivania_mansarda_01_battery
    binary_sensor.faretto_pc_2_update_available
    sensor.interruttore_mansarda_01_battery
    binary_sensor.faretto_pc_1_update_available
**

Ho provato a leggere quanto riportato nel link, ma non ci ho capito nulla, avete una idea di come risolvere?

Grazie

**Comment by KC-inDomus** [comment]
leggi le news indomus

**Reply by snakuzzo** [reply]
è tutto spiegato. Leggi bene.
Non devi risolvere niente tu...devi aspettare che le integrazioni che sfruttano autodiscovery mqtt, con il quale vengono generati i device, rilascino aggiornamenti.
Fine.
